src/autopas/tuning/AutoTuner.cpp

```diff
@@ -95,11 +95,11 @@ bool AutoTuner::tuneConfiguration() {
   auto restoreConfigQueueIfEmpty = [&](const auto &configQueueBackup, const TuningStrategyOption &stratOpt) {
     if (_configQueue.empty()) {
       _configQueue = configQueueBackup;
+      AutoPasLog(WARN, "ConfigQueue wipe by {} detected! Resetting to previous state: (Size={}) {}",
+                 stratOpt.to_string(), _configQueue.size(),
+                 utils::ArrayUtils::to_string(_configQueue, ", ", {"[", "]"},
+                                              [](const auto &conf) { return conf.toShortString(false); }));
     }
-    AutoPasLog(WARN, "ConfigQueue wipe by {} detected! Resetting to previous state: (Size={}) {}", stratOpt.to_string(),
-               _configQueue.size(), utils::ArrayUtils::to_string(_configQueue, ", ", {"[", "]"}, [](const auto &conf) {
-                 return conf.toShortString(false);
-               }));
```

Comment:
I noticed that regardless of whether the queue is empty or not, the warning `AutoPasLog(WARN, “ConfigQueue...` is issued. Is that correct? I have now changed this so that the warning is only printed if the `_configQueue` is empty.

Reply:
Classic off by 1 error. The closing bracket was one line too early ;)
Good you found that. Yesterday I was annoyed about this message being so spammy so I put it on my debug todo list for next week.